packages/office-ui-fabric-react/src/components/Rating/Rating.base.tsx

```diff
@@ -142,14 +155,12 @@ export class RatingBase extends React.Component<IRatingProps, IRatingState> {
       >
```

Comment:
Is this component a slider at the end of the day? If so, it should be marked up as such: [https://www.w3.org/TR/wai-aria-1.1/#slider](https://www.w3.org/TR/wai-aria-1.1/#slider)

Reply:
I don't think it's close enough to ARIA's definition of a slider:

> A slider represents the current value and range of possible values via the size of the slider and position of the thumb.

The [few](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Role_Img#Examples) [examples](https://www.w3.org/WAI/tutorials/forms/custom-controls/) I find don't use `slider` anywhere. If anything, it seems like we should be using `role` of `img` instead of `presentation`.

Reply:
role img is almost always indicative of a static image. I found the right role, radioGroup: [https://www.w3.org/TR/wai-aria-1.1/#radiogroup](https://www.w3.org/TR/wai-aria-1.1/#radiogroup) and each of the items should have role radio: [https://www.w3.org/TR/wai-aria-1.1/#radio](https://www.w3.org/TR/wai-aria-1.1/#radio)

---
In reply to: [328340606](https://github.com/OfficeDev/office-ui-fabric-react/pull/10625#discussion_r328340606) [](ancestors = 328340606)